static/styles/settings.css

```diff
@@ -276,9 +276,25 @@ td .button {
         z-index: 1;
     }
 
+    #id_realm_authentication_methods {
+        width: 20%;
+
+        th,
+        td {
+            border-collapse: collapse;
+        }
+
+        td {
+            border: none;
+            border-top: none;
+            border-bottom: none;
```

Comment:
We don't require these `border-top` and `border-bottom` css, `border: none` is enough.

Reply:
okaay.